src/Model/GroundWaterTransport/gwt-mst.f90

```diff
@@ -855,11 +871,11 @@ subroutine mst_calc_csrb(this, cnew)
       csrb = DZERO
       if (this%ibound(n) > 0) then
         distcoef = this%distcoef(n)
-        if (this%isrb == 1) then
+        if (this%isrb == SORPTION_LINEAR) then
```

Comment:
Not important, but just to mention,  you could use a case statement here (and elsewhere) instead to improve clarity.  

```
select case (this%isrb)
case (SORPTION_LINEAR)
...
case (SORPTION_FREUND)
...
case (SORPTION_LANG)

end select
```

Reply:
Thanks for reminding me of this use case for an `ENUMERATE`, implemented in 6f0d6bb